lib/workers/global/config/parse/index.ts

```diff
@@ -49,10 +50,16 @@ export async function parseConfigs(
   }
 
   if (!config.privateKeyOld && config.privateKeyPathOld) {
-    config.privateKey = await readSystemFile(config.privateKeyPathOld, 'utf8');
+    config.privateKeyOld = await readSystemFile(
```

Comment:
is this a different bug? because previously `config.privateKey` was assigned.

Reply:
Yes, it's a bug I noticed at the same time. OK to fix in one PR? It's intentional